pyproject.toml

```diff
@@ -12,6 +12,12 @@ line-length = 88
 [tool.ruff.lint.flake8-bugbear]
 extend-immutable-calls = ["fastapi.Depends", "fastapi.params.Depends","typer.Option"]
 
+[tool.ruff.lint.isort]
+known-first-party = ["corebackend"]
+
+[tool.isort]
+known_first_party = ["corebackend"]
```

Comment:
What do these do?

Reply:
Tells isort that "corebackend" is our library so it's sorts it correctly. Usually it does a good job by itself but was failing for me locally for some reason.

https://pycqa.github.io/isort/docs/configuration/options.html#known-third-party

Reply:
shouold it have been `core_backend`?

Reply:
ugh. Yeah :/ I'll fix in. the next one